### ISSUE TITLE
Support for Azure Bicep

### ISSUE THREAD

**Issue by chrisjaimon2012** [bridgecrewio/checkov#1989]
Can the checkov scanning library include support for Azure Bicep

https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview

**Comment by JamesWoolfenden** [comment]
As Bicep compiles into ARM template there is already some support for this.

**Comment by SebastianSchuetze** [comment]
Just tested it. In your pipeline you call your main bicep file with the cli and give an output path like this:
```
bicep build ./templates/main.bicep --outdir .dist
```

then you can tell checkov to scan that .dist folder. So I think it is not needed for checkov, because bicep is a superset of ARM templates. Meaning what is wrong in bicep is first handled by the transpiler (syntax and some semantics) but the policies that you would break with checkov would be identical for both.

long story short. Just transpile bicep to ARM templates and check the ARM templates.

**Comment by chrisjaimon2012** [comment]
Thanks a lot @SebastianSchuetze  
I'll try it out. Let me know if you need any help in getting the checkov documentations updated for this (if needed)
